packages/cli/lib/commands/install.js

```diff
@@ -53,9 +60,21 @@ async function install (versionNum) {
 export default install
 
 export async function installFromWeb (versionNum, printMessage = true) {
-  const zipPath = TfvmFS.getPath(TfvmFS.tfVersionsDir, `v${versionNum}.zip`)
-  const newVersionDir = TfvmFS.getPath(TfvmFS.tfVersionsDir, 'v' + versionNum)
-  const url = `https://releases.hashicorp.com/terraform/${versionNum}/terraform_${versionNum}_${TfvmFS.architecture}.zip`
+  const settingsObj = await getSettings()
+
+  let url
+  let zipPath
+  let newVersionDir
+
+  if (settingsObj.useOpenTofu) {
```

Comment:
OpenTofu only exists at 1.6 and beyond. So I think this needs to address being able to use actual terraform for versions that existed before OpenTofu.